src/pudl/output/pudltabl.py

```diff
@@ -932,6 +938,73 @@ def mcoe(
             )
         return self._dfs['mcoe']
 
+    def gens_mega_eia(self, update=False):
+        """
+        Generate and return a generators table with ownership integrated.
+
+        Args:
+            update (boolean): If true, re-calculate the output dataframe, even
+                if a cached version exists. Defualt is `False`.
+
+        Returns:
+            pandas.DataFrame: a table of all of the generators with identifying
+            columns and data columns, sliced by ownership which makes
+            "total" and "owned" records for each generator owner. The "owned"
+            records have the generator's data scaled to the ownership percentage
+            (e.g. if a 100 MW generator has a 75% stake owner and a 25% stake
+            owner, this will result in two "owned" records with 75 MW and 25
+            MW). The "total" records correspond to the full plant for every
+            owner (e.g. using the same 2-owner 100 MW generator as above, each
+            owner will have a records with 100 MW).
+        """
+        if update or self._dfs['gens_mega_eia'] is None:
+            self._dfs['gens_mega_eia'] = (
+                pudl.analysis.plant_parts_eia.MakeMegaGenTbl(self).execute()
+            )
+        return self._dfs['gens_mega_eia']
+
+    def true_grans_eia(self, update=False):
+        """
+        Generate and return a table .
+
+        Note: This table is entirely intented for use within
+        ``PudlTabl.plant_parts_eia`` and would be difficult to work with
+        outside of that context.
```

Comment:
yeah i was wondering this myself. I mostly set it up this way for debugging because I have very often needed to generate the final output but not these two interim outputs. I think with the `deep_update` argument of `plant_parts_eia` this workflow could stay enabled without having their own output methods.